src/main/java/core/basesyntax/DataValidator.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+import core.basesyntax.exeptions.NotValidDataException;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+
+public class DataValidator {
+    private static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("uuuu-MM-dd");
```

Comment:
do we have a predefined formatter for this purpose?

Reply:
done